nft/models/_sector/aggregators/chains/nft_ethereum_aggregators_gem.sql

```diff
@@ -1,14 +1,21 @@
 {{config(
-    
     schema = 'nft_ethereum',
-    alias = 'aggregators_gem'
-)}}
+    alias = 'aggregators_gem',
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
```

Comment:
Can we also add `incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_time')],` here and below

Reply:
ah yeah you are right ofc. Saw that you added it where it's possible.